provider/pktgen_utils.py

```diff
@@ -18,6 +20,34 @@ def __init__(self, interface=None, dsc=None, runner=None):
         self.dsc = dsc
         self.runner = runner
 
+    def vp_vdpa_bind(self, session_serial):
+        try:
+            LOG_JOB.info("Starting the binding process for Virtio 1.0 network devices.")
+            pci_list = utils_misc.get_pci_id_using_filter("Virtio 1.0 network", session_serial)
+            if not pci_list:
+                raise ValueError("No PCI devices found matching 'Virtio 1.0 network'.")
+
+            pci_id = pci_list[0]
+            if not pci_id.startswith("0000:"):
```

Comment:
This should be fixed with `lspci -D` in VT.

Reply:
Since the pci_id was come from utils_misc.get_pci_id_using_filter, "lspci -D" is not suitable for other test cases.

Reply:
Yes, so that function has a bug, I think every case needs a full ID with domain numbers.

Reply:
I don't think every case needs it. I add parameter domain in function utils_misc.get_pci_id_using_filter.